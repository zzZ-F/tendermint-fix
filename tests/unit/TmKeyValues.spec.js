import { shallowMount } from "@vue/test-utils"
import htmlBeautify from "html-beautify"
import { TmKeyValues } from "@/index"
import { TmKeyValue } from "@/index"

describe("TmKeyValues.vue", () => {
  let wrapper
  beforeEach(async () => {
    wrapper = shallowMount(TmKeyValues, {
      slots: { default: "<div>slot</div>" }
    })
  })

  it("has the expected html structure", () => {
    expect(htmlBeautify(wrapper.html())).toMatchSnapshot()
  })
})
describe("TmKeyValue.vue", () => {
  let wrapper
  beforeEach(async () => {
    wrapper = shallowMount(TmKeyValue, {
      slots: { key: "<div>key</div>", value: "<div>value</div>" }
    })
  })

  it("has the expected html structure", () => {
    expect(htmlBeautify(wrapper.html())).toMatchSnapshot()
  })
})
