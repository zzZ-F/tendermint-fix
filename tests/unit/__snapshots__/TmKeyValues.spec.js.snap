// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TmKeyValue.vue has the expected html structure 1`] = `
<div class="tm-kv">
  <div class="tm-kv-container">
    <div class="tm-kv-value">
      <div>
        value
      </div>
    </div>
    <div class="tm-kv-key">
      <div>
        key
      </div>
    </div>
  </div>
</div>
`;

exports[`TmKeyValues.vue has the expected html structure 1`] = `
<div class="tm-kvs">
  <div>
    slot
  </div>
</div>
`;
